src/hotspot/share/opto/memnode.cpp

```diff
@@ -3014,9 +3014,15 @@ uint LoadStoreNode::ideal_reg() const {
 }
 
 bool LoadStoreNode::result_not_used() const {
```

Comment:
Add comment to this method explaining what it is for and cases when it returns `true` or `false`.